src/Components/CountryCard.js

```diff
@@ -20,7 +20,7 @@ const CountryTiles = () => {
 
   const searchHandler = (n) => {
     const countries = n.target.value;
-    const list = country.filter((e) => e.countryName.includes(countries));
+    const list = country.filter((e) => e.countryName.includes(countries.toLowerCase()));
```

Comment:
- [ ] Kindly check on your search method, currently, I can't search a nation by typing the country's full name but when I delete the first letter of a nation I can search a country below is a picture demonstration:

Writing Full Name:
![image](https://user-images.githubusercontent.com/61408860/175312174-35b38a67-fefd-428a-8f6c-a15ee8278005.png)

Writing All Letters Except the First Letter
![image](https://user-images.githubusercontent.com/61408860/175312452-e4211893-4e85-4cee-8fd3-6082fac1fe36.png)